### ISSUE TITLE
Please stop squatting crates.io package names

### ISSUE THREAD

**Issue by piedoom** [cuimoman/dufu#1]

**Comment by Anders429** [comment]
Agreed. What you're doing is against the ToS of crates.io, as I understand it, but even if it weren't it's just a generally terrible thing to do. It actively hurts the community by taking up names that might otherwise be used for some beneficial purpose.
